docs/gettingStarted/setupCustomSchemeAlchemy.md

```diff
@@ -1 +1,120 @@
+  4. integration test (optional): `test/integration/MyContractName.spec.ts`
+
+
+  5. Add your contract to `ops/mappings.json`. Under the JSON object for the network your contract is located at, under the `"mappings"` JSON array, add the following.
+
+    1. If your contract information is in the `migration.json` file specified (default is the file under `@daostack/migration` folder, as defined in the `ops/settings.js` file)
+
+            {
+               "name": "<contract name as appears in `abis/arcVersion` folder>",
+               "contractName": "<contract name as appears in migration.json file>",
+               "dao": "<section label where contract is defined in migration.json file (base/ dao/ test/ organs)>",
+               "mapping": "<contract name from step 2>",
+               "arcVersion": "<contract arc version>"
+            },
+
+    2. If your contract does not appear in the migration file:
+
+            {
+               "name": "<contract name as appears in `abis/arcVersion` folder>",
+               "dao": "address",
+               "mapping": "<MyContractName>",
+               "arcVersion": "<contract arc version under which the abi is located in the `abis` folder>",
+               "address": "<the contract address>"
+            },
+
+  6. (Optionally) add a deployment step for your contract in `ops/migrate.js` that will run before testing.
+
+#### Deploy subgraph and start graph node locally
+
+  1. Update `.env` file in `subgraph` directory
+
+        # Following are example values please change for customization
+        network="rinkeby"
+        subgraph="daostack"
+        postgres_password="letmein"
+        ethereum_node="https://rinkeby.infura.io/v3/e0cdf3bfda9b468fa908aa6ab03d5ba2"
+
+  2. Start `graph-node` in docker for your testnet
+
+        npm run docker:run-rinkeby 
+
+  3. Add your DAO details and deploy subgraph
+
+        touch daos/rinkeby/<DAO-Name>.json
+
+        # Add your DAO details in <DAO-Name.json> file
+        # refer to any of existing file in daos/rinkeby folder
+        # you can skip argument to deploy if using default `migration.json` from `@daostack/migration`
+
+        npm run deploy '{  "migrationFile" : "../migration.json" }'
+        
+  4. Go back to directory `alchemy` and Update `docker-compose.yml`
+
+        - remove link to `graph-node` in service `alchemy`
+        - remove services `graph-node`, `ipfs`, `postgres4graphnode` and `ganache`
+
+  5. Update `webpack.docker.config.js`, add following process variables
+
+        'ARC_GRAPHQLHTTPPROVIDER': JSON.stringify('http://127.0.0.1:8000/subgraphs/name/daostack'),
+        'ARC_GRAPHQLWSPROVIDER': JSON.stringify('ws://127.0.0.1:8001/subgraphs/name/daostack'),
+        'ARC_IPFSPROVIDER': JSON.stringify('localhost')
+
+      NOTE: If you changed name of `subgraph` while setting up `.env` in `step 1` then change it in this step accordingly
+      
+  6. Build and run
+          docker-compose up -d
```

Comment:
This will probably try to start another graph-node server opn ports 8000 and 8001, and so will fail. 
Again, I'd run `npm run start` and not bother with the docker setup in this case..